### PR TITLE
bsdlib: version 0.8.0

### PR DESCRIPTION
**Pull request by evenl**
Library version: 866aa7e4db6

* Fixed the issue with stalled TLS handshake.
* Fixed the issue with TLS connection where `nrf_connect` hangs.
* Fixed the issue of `nrf_sendto` timeout not working in some cases.
* Updated the documentation to reflect that NRF_SO_CHIPER_IN_USE
  is not currently supported.
* Fixed the issue of missing AT socket and POLLIN events.
* Added support for PDN authentication parameters.
* Added flushing of the GNSS socket queue when command is issued.
* Added support for GPS low accuracy use case.

Manifest update: https://github.com/nrfconnect/sdk-nrf/pull/3055

### CODE REVIEW

**Review by lemrey — CHANGES_REQUESTED** [review]
Hang on, something seems off with the carrier library.

File: bsdlib/doc/CHANGELOG.rst [hunk 1]
```diff
@@ -5,12 +5,25 @@ Changelog
...
+
+* Fixed issue with stalled TLS handshake.
+* Fixed issue with TLS connection where connect() hangs.
+* Fixed issue with sendto timeout not working in some cases.
+* Updated the documentation to reflect that NRF_SO_CHIPER_IN_USE is not
+  currently supported.
+* Fixed issue with the AT socket and POLLIN events not happening.
+* Added support for PDN authentication parameters.
+* Added flushing of the GNSS socket queue if the stop command is issued.
+* Added support for GPS low accuracy use-case.
```
Comment (umapraseeda) [inline bsdlib/doc/CHANGELOG.rst:19]:
```suggestion
* Fixed the issue with stalled TLS handshake.
* Fixed the issue with TLS connection where :cpp:func:`nrf_connect` hangs.
* Fixed the issue of sendto timeout not working in some cases.
* Updated the documentation to reflect that NRF_SO_CHIPER_IN_USE is not currently supported.
* Fixed the issue of missing AT socket and POLLIN events.
* Added support for PDN authentication parameters.
* Added flushing of the GNSS socket queue if the stop command is issued.
* Added support for GPS low accuracy use case.
```

File: bsdlib/doc/CHANGELOG.rst [hunk 2]
```diff
@@ -5,12 +5,25 @@ Changelog
...
+
 bsdlib 0.7.9
 ************
 
 * Fixed an issue introduced with the TLS server support that made :cpp:func:`nrf_connect` hang forever.
 
-bsdlib 0.7.8
+bdli 0.7.8
```
Comment (umapraseeda) [inline bsdlib/doc/CHANGELOG.rst:26]:
```suggestion
bsdlib 0.7.8
```

File: bsdlib/doc/CHANGELOG.rst (hunk 1 above)
Comment (lemrey) [reply bsdlib/doc/CHANGELOG.rst]: Updated

File: bsdlib/doc/CHANGELOG.rst (hunk 2 above)
Comment (lemrey) [reply bsdlib/doc/CHANGELOG.rst]: Updated

**Comment by lemrey** [comment]
@umapraseeda I have updated based on your suggestions, please revist.

**Review by rlubos — APPROVED** [review]

**Comment by umapraseeda** [comment]
> @umapraseeda I have updated based on your suggestions, please revist.

Thanks.. LGTM!

**Comment by lemrey** [comment]
Rebased

**Review by umapraseeda — APPROVED** [review]

**Review by lemrey — APPROVED** [review]

**Comment by lemrey** [comment]
Waiting for https://github.com/nrfconnect/sdk-nrf/pull/3055

**Pull request merged**